src/main/kotlin/no/nav/bidrag/bidragskalkulator/consumer/BidragPersonConsumer.kt

```diff
@@ -8,20 +9,37 @@ import no.nav.bidrag.transport.person.MotpartBarnRelasjonDto
 import no.nav.bidrag.transport.person.PersonDto
 import no.nav.bidrag.transport.person.PersonRequest
 import org.springframework.beans.factory.annotation.Qualifier
-import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Service
 import org.springframework.web.client.HttpServerErrorException
 import org.springframework.web.client.RestTemplate
+import org.springframework.web.util.UriComponentsBuilder
 import java.net.URI
 
-@Service
+@Service("bidragPersonConsumer")
 class BidragPersonConsumer(
-    @Value("\${BIDRAG_PERSON_URL}") bidragPersonUrl: URI,
+    val bidragPersonConfig: BidragPersonConfigurationProperties,
     @Qualifier("azure") restTemplate: RestTemplate
-) : AbstractRestClient(restTemplate, "bidrag-person") {
+) : AbstractRestClient(restTemplate, "bidrag.person") {
+
+    init {
+        check(bidragPersonConfig.url.isNotEmpty()) { "bidrag-person url mangler i konfigurasjon" }
+        check(bidragPersonConfig.hentMotpartbarnrelasjonPath.isNotEmpty()) { "hentMotpartbarnrelasjonPath mangler i konfigurasjon" }
```

Comment:
Skulle vi hatt `bidrag-person`-prefiks her og?
```suggestion
        check(bidragPersonConfig.hentMotpartbarnrelasjonPath.isNotEmpty()) { "bidrag-person hentMotpartbarnrelasjonPath mangler i konfigurasjon" }
```
Evt slik for å vise hierarki?
```suggestion
        check(bidragPersonConfig.hentMotpartbarnrelasjonPath.isNotEmpty()) { "bidrag.person.hentMotpartbarnrelasjonPath mangler i konfigurasjon" }
```